main.go

```diff
@@ -108,6 +119,10 @@ func main() {
 		return
 	}
 
+	if(didSetVerboseAlready()){
+		panic(fmt.Errorf("can't use both -v and -verbose flags, must use only one"))
```

Comment:
s/`can't use`/`cannot set`

and would drop `, must use only one`, since it's not a must to set either in fact